.goreleaser.yml

```diff
@@ -0,0 +1,61 @@
+before:
+  hooks:
+    - make release
+builds:
+  -
+    id: capp
+    env:
+      - CGO_ENABLED=0
+    binary: manager
```

Comment:
Will this create the artifacts in the repo root directory? Or in `dist/`? I don't see anything in the config that pushes it into `dist`, but maybe that is the default?

Also, how do we distinguish between one build and another, so we don't accidentally put the darwin-amd64 binary in the linux-arm64 image? The current `make build` target creates `bin/manager-<GOOS>-<GOARCH>`. Can we reuse that structure? Or is goreleaser limited from doing that? I imagine not, with those go text templates I see below, but I don't know.

Reply:
Everything that gets created by goreleaser is in `dist`

Reply:
And when it builds, how does it differentiate between the multiple binaries? It looks like it names them all `manager`, but there are multiple? Does it use the `manager` tag just as a tag, and automatically append the right arch/etc.? If so, how does `Dockerfile.goreleaser` get the right file? Or does it actually manipulate the dockerfile before running it?